modules/op-resolver-git-config-scm.adoc

```diff
@@ -0,0 +1,66 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-git-config-scm_{context}"]
+= Configuring the Git resolver for the authenticated SCM API
+
+For the authenticated SCM API, you must set the configuration for the authenticatred Git connection.
+
+You can use Git repository providers that are supported by the link:https://github.com/jenkins-x/go-scm/tree/main/scm/driver[`go-scm` library]. Not all `go-scm` implementations have been tested with the Git resolver, but the following providers are known to work:
```

Comment:
We usually avoid linking to GitHub (if you need to, you need to get approval from a support/product experience contact - ask Ashleigh or Brian, not sure if your contact is different than core OCP's)